Sort stage1 manifest files

glevand opened the pull request:
To ease maintenance let's keep the stage1 manifest files sorted.  Adds a script `sort-stage1-manifests.sh`, and sorts the usr_from_kvm files.  The usr_from_coreos files were sorted in https://github.com/rkt/rkt/commit/e70889eea122e73afd94cc2598f87b4919328b30 (stage1: upgrade to systemd-v233 from CoreOS 1478.0.0).

cc: @alban

alban commented:
The script worked fine when I tried it last time.

@glevand could you add a section at the end of the documentation [update-coreos-stage1.md](https://github.com/rkt/rkt/blob/master/Documentation/devel/update-coreos-stage1.md) to say to use that script?

glevand commented:
Updated script to remove duplicate and empty lines.
Added requested comment to [update-coreos-stage1.md](https://github.com/rkt/rkt/blob/master/Documentation/devel/update-coreos-stage1.md).
